x-pack/elastic-agent/pkg/agent/transpiler/rules.go

```diff
@@ -429,6 +436,117 @@ func InjectIndex(indexType string) *InjectIndexRule {
 	}
 }
 
+// InjectStreamProcessorRule expect target to be a collection of fields including
+// _meta_index map with dataset, index and namespace keys defined.
```

Comment:
I think this comment need to be changed since we do not have the `_meta_index` present in the code?